.github/workflows/tests.yml

```diff
@@ -281,3 +278,45 @@ jobs:
         uses: re-actors/alls-green@release/v1
         with:
           jobs: ${{ toJSON(needs) }}
+
+  build:
+    name: Canary Build
+    needs: [analyze]
+    # only build canary build iff
+    # - prior steps succeeded,
+    # - this is the main repo, and
+    # - we are on the main (or feature) branch
+    if: >-
+      success()
+      && !github.event.repository.fork
+      && (
+        github.head_ref == "main"
+        || startsWith(github.head_ref, "feature/")
+      )
```

Comment:
ah, good catch

what if instead of using the `dev` label we use `${{ github.ref_name }}`?

```
conda install -c "conda-canary/label/main" conda
conda install -c "conda-canary/label/feature/plugins" conda
```

or we can keep the dev label for main (e.g, `${{ github.ref_name == 'main' && 'dev' || github.ref_name }}`):

```
conda install -c "conda-canary/label/dev" conda
conda install -c "conda-canary/label/feature/plugins" conda
```